src/content/en/2021/seo.md

```diff
@@ -1,20 +1,723 @@
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1099454676&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats.sql"
+   )
+}}
+
+### Meta Description Tag
+
+The `<meta name="description>` tag does not directly impact rankings. However, it may appear as the page description on the SERP.
+
+In 2021:
+
+- The median desktop and mobile page `<meta name="description>` tag contained 20 and 19 words, respectively.
+- The median desktop and mobile page `<meta name="description>` tag contained 138 and 127 characters, respectively.
+- 10% of desktop and mobile pages had `<meta name="description>` tags containing 35 words.
+- 10% of desktop and mobile pages had `<meta name="description>` tags containing 232 and 231 characters, respectively.
+
+These numbers are relatively unchanged from last year.
+
+{{ figure_markup(
+   image="meta-word-counts.png",
+   caption="Number of words used in meta descriptions.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=2013621429&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats.sql"
+   )
+}}
+
+{{ figure_markup(
+   image="meta-character-counts.png",
+   caption="Number of characters used in meta descriptions.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=971210715&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats.sql"
+   )
+}}
+
+### Images
+
+Images can directly and indirectly impact SEO as they impact image search rankings and page performance.
+
+- 10% of pages have two or fewer `<img>` tags. That's true of both desktop and mobile.
+- The median desktop page has 21 `<img>` tags while the median mobile page has 19 `<img>` tags.
+- 10% of desktop pages have 83 or more `<img>` tags. 10% of mobile pages have 73 or more `<img>` tags.
+
+These numbers have changed very little since 2020.
+
+{{ figure_markup(
+   image="number-of-images-per-page.png",
+   caption="Number of images on each page.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1314615789&format=interactive",
+   sheets_gid="1483073708",
+   sql_file="image-alt-stats.sql"
+   )
+}}
+
+The `<img>` ALT attribute helps explain image content and impacts [accessibility](https://almanac.httparchive.org/en/2021/accessibility).
+
+Note that missing ALT attributes may not indicate a problem. Pages may include extremely small or blank images which don't require an ALT attribute.
```

Comment:
"or repetitive of nearby content images" I'm going to need you to explain that one.

Reply:
I presume he means things like:

```html
<img src="Alert icon" alt=""> Alert: This is an important message</img>
```

In that case there is no need for an alt on the image since it's covered by the text.

Don't think we need to get into this level of detail here so resolving this with no change.